From 8474ba255573bc57d62eaa887f1079e214dbec18 Mon Sep 17 00:00:00 2001
From: Giovanni Campagna <gcampagna@src.gnome.org>
Date: Thu, 3 Nov 2011 10:02:03 +0100
Subject: [PATCH 2/3] NetworkMenu: fix regression in access-point-removed

When changing _findNetwork with _findExistingNetwork, I changed
the return value to avoid searching twice for the access point,
and changed some names. I forgot to update all points where those
names were used.

https://bugzilla.gnome.org/show_bug.cgi?id=663278
---
 js/ui/status/network.js |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/js/ui/status/network.js b/js/ui/status/network.js
index 966e872..aa9b277 100644
--- a/js/ui/status/network.js
+++ b/js/ui/status/network.js
@@ -1322,7 +1322,7 @@ NMDeviceWireless.prototype = {
                     this._overflowItem = null;
                 }
             }
-            this._networks.splice(pos, 1);
+            this._networks.splice(res.network, 1);
 
         } else if (apObj.item)
             apObj.item.updateAccessPoints(apObj.accessPoints);
-- 
1.7.7.3

